bitnami/nginx/values.yaml

```diff
@@ -110,6 +110,27 @@ ingress:
 #     }
 #   }
 
+## Liveness Probe
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/
+##
+livenessProbe:
+  httpGet:
+    path: /
+    port: http
+  initialDelaySeconds: 30
+  timeoutSeconds: 5
+  failureThreshold: 6
+
+## Readiness Probe
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/
+readinessProbe:
+  httpGet:
+    path: /
+    port: http
+  initialDelaySeconds: 5
+  timeoutSeconds: 3
+  periodSeconds: 5
```

Comment:
What about other values such as the one below?

```yaml
    successThreshold:
    failureThreshold:
```

Reply:
My goal was to make the probes configurable while changing as little as possible. I have no problem with adding some new functionality if that is the desire.

What would you like the defaults to be for the new fields?

Alternatively, something I've done before is to use `{{ toYaml .Values.livenessProbe }}`, which lets the user configure whatever parameters they want. That's definitely a more radical change but it gives the user more options.

Reply:
I agree it's more flexible for the user. I'd follow that approach.